lib/ansible/module_utils/aws.py

```diff
@@ -0,0 +1,111 @@
+# This code is part of Ansible, but is an independent component.
```

Comment:
Though if things are a literal copy of what's in ec2.py (as CloudRetry is) I'd rather use an import so we don't duplicate all the code.